### PR TITLE
Add IRC alerts for third party services

### PR DESCRIPTION
**Pull request by helfi92**
IRC alerts for:
* Sentry
* Signal FX
* Papertrail

### CODE REVIEW

File: bin/hubot.cmd [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@echo off
+
+call npm install
```
Comment (eliperelman) [inline bin/hubot.cmd]: Why npm and not yarn?

File: bin/hubot.cmd [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+SETLOCAL
+SET PATH=node_modules\.bin;node_modules\hubot\node_modules\.bin;%PATH%
```
Comment (eliperelman) [inline bin/hubot.cmd]: Is this file for Windows? We wouldn't need that if we're launching in Heroku or similar.

File: scripts/index.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    if (!ircRooms) {
+      return;
+    }
+
+    const rooms = ircRooms.split(',[\s]*');
+
+    rooms.forEach(room => robot.messageRoom(room, message));
+  };
+
+  robot.router.post('/hubot/sentry', function(req, res) {
```
Comment (eliperelman) [inline scripts/index.js]: Arrow function.

File: scripts/index.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    const data = alertFromRequest(req);
+    const message = `[Sentry] Alert from ${data.project} ${data.project_name}\n` +
```
Comment (eliperelman) [inline scripts/index.js]: Consider using a single template string and something like the common-tags package's `stripIndents` in the future.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  "dependencies": {
+    "coffee-script": "^1.12.6",
+    "hubot": "^2.19.0",
+    "hubot-diagnostics": "0.0.2",
+    "hubot-help": "^0.2.2",
+    "hubot-irc": "^0.4.0"
+  },
+  "scripts": {
+    "start": "bin/hubot",
+    "test": "mocha test/*_test.js",
```
Comment (eliperelman) [inline package.json]: Not that big of a deal, but why not Neutrino to unify these settings?

**Review by eliperelman — APPROVED** [review]

File: scripts/index.js (hunk 4 above)
Comment (helfi92) [reply scripts/index.js]: @eliperelman any idea how to create multiline IRC messages. This currently sends a new message for each line. Not the best UX.

Comment (eliperelman) [reply scripts/index.js]: I'm not sure. Maybe there is a way to put all the relevant data in a single line, with a link to details?

File: bin/hubot.cmd (hunk 1 above)
Comment (helfi92) [reply bin/hubot.cmd]: This file was auto-generated when running the hubot generator so I didn't touch it. Changing it to `yarn` sgtm.

File: bin/hubot.cmd (hunk 2 above)
Comment (helfi92) [reply bin/hubot.cmd]: Yeah, for Windows. I will delete it.

**Pull request merged**